### ISSUE TITLE
Atypical scenario leads to a runtime error

### ISSUE THREAD

**Issue by BackOrder** [Shoes3/shoes3#23]
```ruby
Shoes.app(:title => "Causing a Runtime Error!", :width => 450, :height => 250) {
   stack { keypress { |key| info key } }
}
```

**TEST SCENARIO**
* Run the script above using Shoes
* Open Shoes console (alt-/)
* Go back to the script window
* Open Shoes manual (alt-?)
* Go back to the script window
* Press few letters, they should appear in Shoes console
* Close Shoes console

![image](https://cloud.githubusercontent.com/assets/8844578/5677379/96bd7492-97b3-11e4-9ee5-9037078a6304.png)

**Comment by BackOrder** [comment]
Running the test scenario above using *cshoes.exe* gave more information. It seems **Pango** is at fault:

```
(cshoes.exe:3660): Pango-CRITICAL **: pango_win32_font_map_get_font_cache: asser
tion `font_map != NULL' failed
**
Pango:ERROR:pangowin32.c:831:pango_win32_font_finalize: assertion failed: (win32
font->fontmap != NULL)

This application has requested the Runtime to terminate it in an unusual way.
Please contact the application's support team for more information.
```

**REFERENCES**
https://mail.gnome.org/archives/commits-list/2013-February/msg08867.html (might be related)

**Comment by ccoupe** [comment]
Fixed in 3.2.22
